ansible/roles/ooni-backend/tasks/populate_bouncer_base.yml

```diff
@@ -15,13 +15,32 @@
     - "{{ bind_collector_hsdir }}"
     - "{{ bind_web_connectivity_hsdir }}"
 
-- name: Template "{{ bind_bouncer_file  }}"
+- name: Template "{{ bind_bouncer_file }}" with bouncer_base.yaml.j2
   template: >
     src="bouncer_base.yaml.j2"
     dest="{{ bind_bouncer_file }}"
     backup="yes"
+    unsafe_writes="true"
+  when: ( oonibackend_bundle is defined and oonibackend_bundle == True )
+  register: bouncer_config_file
 
-- name: Fetch backend related config files and HS keys
+- name: Template "{{ bind_bouncer_file }}" with "{{ bouncer_base_template }}"
+  template: >
+    src="{{ bouncer_base_template }}"
+    dest="{{ bind_bouncer_file }}"
+    backup="yes"
+    unsafe_writes="true"
+  when: ( oonibackend_bundle is undefined or oonibackend_bundle == False )
```

Comment:
> On the other hand the casting to a default value is used in many more places:

> $ grep -rn 'default(' ansible | wc -l
>      24

In your [comment](https://github.com/TheTorProject/ooni-sysadmin/pull/79#discussion_r94381954) you mentioned that I should instead cast to bool.
```
 grep -rn 'bool' ansible | wc -l
 0
 ```
 Also the default filter is only used in templates:
 ```
 grep --exclude-dir=templates -rn 'default(' ansible | wc -l
 0
 ```

 I can change all the conditionals of this repository to your preferred syntax pattern but I believe this is not related and should not be addressed in this PR.